app/public/js/player/playerCtrl.js

```diff
@@ -358,4 +360,85 @@ app.controller('PlayerCtrl', function (
     });
 
 
+    var server = http.createServer(function (req, res) {
```

Comment:
Move all this new code to a new folder and file e.g `app/public/js/control/control.js` `app/public/view/control/control.html`